app/controllers/resources_controller.rb

```diff
@@ -19,8 +19,61 @@ def search
     render json: ResourcesPresenter.present(result)
   end
 
+  def create
+    resources_params = clean_resources_params
+    resources = resources_params.map { |r| Resource.new(r) }
```

Comment:
Do we need this route to accept a list of resources? I think this is a little bit unexpected because the standard Rails CRUD routes only allow you to create one object per request.